Darya Kozlova/cubes_style.css

```diff
@@ -0,0 +1,76 @@
+            body {
+            height: 100%;
+            background: url(img/cubes_background.png) no-repeat;
+            background-size: cover;
+        }
+        
+        .all {
+            display: flex;
+            flex-wrap: wrap;
+            color: white;

```

Comment:
Keep in mind order of properties, group them by type 
1 - layout (position, display)
2 - anything related to box model (width/height/margin/padding)
3 - styling for element (color, background, box-shadow)
4 - anything related to font 
5 - other stuff, it also includes overflow, z-index